src/components/ui/admin/settings/listSlides/PointSlide.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { ReactNode } from 'react';
+import { Flex, Input, AutoComplete, Checkbox } from 'antd';
+import { useMotionValue, Reorder } from 'framer-motion';
+import { Slide } from '~types/settings';
+import { Close } from '~components/ui/icons/close';
+import { useRaisedShadow } from './hooks/useRaisedShadow';
+const { TextArea } = Input;
+
+type PointSlideProps = {
+    slide: Slide;
+    handleRemove: (id: number) => void;
+    imageOptions: {
+        fileDownload: string;
+        label: ReactNode | string;
+        value: string;
+    }[];
+    changeSlide: (id: number, obj: Partial<Slide>) => void;
+};
+
+export const PointSlide: React.FC<PointSlideProps> = ({
+    slide,
+    handleRemove,
+    imageOptions,
+    changeSlide,
+}) => {
+    const y = useMotionValue(0);
+    const boxShadow = useRaisedShadow(y);
+
+    return (
+        <Reorder.Item
+            key={slide.id}
+            value={slide}
+            id={slide.img}
+            style={{ boxShadow, y }}
+        >
+            <Flex className="list-item-inner">
+                <div className="close-button-wrapper">
+                    <button
+                        className="close-button"
+                        onClick={() => handleRemove(slide.id)}
+                    >
+                        <Close width={18} hanging={18} />
+                    </button>
+                </div>
+                <div className="imgbox">
+                    {/* TODO: convert to Image Nextjs */}
```

Comment:
I think it will be okay to use just img if we can add aria-label for accessibility

Reply:
unfortunately aria-label does not solve the problem